test/multicluster/minikube_base_multicluster_test.go

```diff
@@ -134,4 +140,73 @@ func TestKubernetesBasicMultiCluster(t *testing.T) {
 			verifyNuoSQL(t, context.Namespace, admin0, "demo")
 		})
 	})
+
+	testlib.RunOnNuoDBVersionCondition(t, ">=5.0.2", func(version *semver.Version) {
+		t.Run("testDatabaseResync", func(t *testing.T) {
+			testlib.DeployWithContext(t,
+				cluster1Context,
+				func(context *testlib.ClusterDeploymentContext, options *helm.Options) {
+					// delete database controllers and PVCs
+					helm.Delete(t, options, context.DatabaseReleaseName, true)
+					testlib.AwaitNoPods(t, namespaceName, context.DatabaseReleaseName)
+					smHCPvcName := fmt.Sprintf("archive-volume-sm-%s-nuodb-%s-demo-hotcopy-0",
+						context.DatabaseReleaseName, context.ThisCluster.Name)
+					k8s.RunKubectl(t, kubectlOptions, "delete", "pvc", smHCPvcName)
+				},
+			)
+			// negative test: check that the database is not deleted
+			testlib.ExecuteInAllClusters(t, func(context *testlib.ClusterDeploymentContext) {
+				admin0 := fmt.Sprintf("%s-nuodb-%s-0", context.AdminReleaseName, context.ThisCluster.Name)
+				testlib.AwaitDatabaseUp(t, context.Namespace, admin0, "demo", 2)
+				testlib.CheckArchives(t, namespaceName, admin0, "demo", 1, 0)
+				db, err := testlib.GetDatabaseE(t, namespaceName, admin0, "demo")
+				require.NoError(t, err)
+				require.NotEqual(t, "TOMBSTONE", db.State)
+			})
```

Comment:
Is this actually a negative test? I consider a negative test to be checking that bad input is correctly rejected.

Reply:
I called it "negative" because the database should not be removed automatically as not all conditions are met. In this case, the domain state is the input so according to your description it is not testing a "bad" input.